queries.sql

```diff
@@ -117,4 +117,84 @@ FROM owners o
 LEFT JOIN animals a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY total_animals DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+
+
+-- Questions 
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name 
+FROM animals a 
+JOIN visits v ON a.id = v.animals_id
+JOIN vets vet ON v.animals_id=vet.id
+WHERE vet.name = 'Vet William Tatcher '
+ORDER BY v.visits_date;
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(*)
+FROM vets v 
+JOIN visits vi ON v.id = vi.vets_id
+WHERE v.name = 'Vet Stephanie Mendez ';
```

Comment:
great you are so close , this query will return total number of visits including redandancies , to correct you can use 
`SELECT COUNT(DISTINCT a.id)
FROM vets v
JOIN visits vi ON v.id = vi.vets_id
JOIN animals a ON vi.animals_id = a.id
WHERE v.name = 'Vet Stephanie Mendez';`

refer to this article:
[https://www.commandprompt.com/education/how-do-i-count-unique-values-in-postgresql/](https://www.commandprompt.com/education/how-do-i-count-unique-values-in-postgresql/)